MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -86,7 +86,7 @@ Note: Backend criteria apply to modules, shared backend libraries, and edge modu
   * -_note: read more at https://github.com/folio-org/okapi/blob/master/okapi-core/src/main/raml/ModuleDescriptor.json_
 * Module includes executable implementations of all endpoints in the provides section of the Module Descriptor
 * Environment vars are documented in the ModuleDescriptor (5, 11)
-  * -_note: read more at [https://wiki.folio.org/pages/viewpage.action?pageId=65110683](https://wiki.folio.org/pages/viewpage.action?pageId=65110683)_
+  * -_note: read more at [https://folio-org.atlassian.net/wiki/spaces/SYSOPS/pages/2097733/Change+Environment+Variables+of+a+Module](https://folio-org.atlassian.net/wiki/spaces/SYSOPS/pages/2097733/Change+Environment+Variables+of+a+Module)_
```

Comment:
See my comment about the correct(?) page.  Also, this notation is redundant if the link text and URL are identical